### DISCUSSION TITLE
Release of V3?

### DISCUSSION THREAD

**Discussion by LordNex** [dwainscheeren/dwains-lovelace-dashboard#443] [Q&A] [answered]
I just did a complete rebuild and upgrade of my entire network including home assistant and as i was reinstalling some features came upon your dashboard. This would be great as I've always created them manually in Lovelace and took a lot of time to get things tweaked out properly.

Anyway, Id like to start fresh and new and seen on your webpage that 3.0 looks to be available. As far as the web page for documentation for 3.0 has an install but only 2.0.7 is available. Even when i looked at your migration from 2.x to 3.x, i cannot seem to get the newer version to display in HACS. I came here to try the manual install and from what I can tell it is out yet.

So is there an ETA on when 3.0 will be released or did i miss something during the install? If its going to be released soon, id prefer to just start at that level instead of putting in all the work reworking the config files if I can. Thanks in advance!

**Comment by dwainscheeren** [comment, marked as answer]
3.0 is in beta available on my Discord server and will be public available this or next week.

**Reply by LordNex** [reply]
Ok cool. I'll pop over there and see if i can find it. Thanks

**Reply by JJrios21** [reply]
When will V3 be available?

**Reply by dwainscheeren** [reply]
@JJrios21 its available

**Comment by LordNex** [comment]
I hopped on the discord site but could get anyone that was online to answer. Any suggestions? I'd really like to start this install with a fresh new desktop and abilities. But wondering if there's a place I can download prior to release, or if I just going to have to wait until the office release gets dropped. Thanks

**Reply by dwainscheeren** [reply]
Are you on the right Discord server? There are hundres of people online and you always get an answer within maximum 2 hours.

**Comment by dwainscheeren** [comment]
https://github.com/dwainscheeren/dwains-lovelace-dashboard/releases/tag/v3.0.0

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
